src/diff/amfGraphDifferencer.ts

```diff
@@ -269,13 +284,17 @@ export function parseNodeDelta(
   delta: AmfGraphDeltaTypes.ArrayElementDelta<AmfGraphTypes.Node>,
   deltaType: DeltaType
 ): NodeChanges {
-  let nodeChanges;
   const node = delta[0];
   const id = node[AmfGraphTypes.KEY_NODE_ID];
   const type = node[AmfGraphTypes.KEY_NODE_TYPE];
+  if (!isNodeTypeDefined(type)) {
+    ramlToolLogger.debug(`Ignoring changes to the node with no type: ${id}`);
+    return undefined;
```

Comment:
```suggestion
    return;
```
`return` implicitly returns `undefined`.